app/page/RoutePage.js

```diff
@@ -86,34 +89,68 @@ class RoutePage extends React.Component {
     }
   }
 
+  isMapFullscreen = () => {
+    if (typeof window !== 'undefined' && supportsHistory()) {
+      const state = this.context.location.state;
+      return state && state.fullscreenMap;
+    }
+
+    return this.state && this.state.fullscreenMap;
+  };
+
+  toggleFullscreenMap = () => {
+    if (supportsHistory()) {
```

Comment:
check for supportsHistory is not needed anymore as support for IE 9 was dropped, so this.state. fullscreenMap can be removed